langchain/embeddings/base.ts

```diff
@@ -1,4 +1,6 @@
 export abstract class Embeddings {
+  abstract numDimensions: number;
```

Comment:
This doesn't exist in the python library, but it felt right? Happy to change however

Reply:
depends on how we do it imo. see comment below